.changelog/2928.breaking.md

```diff
@@ -0,0 +1,24 @@
+Change staking account ids/addresses to truncated hash of the public key
+
+Previously, staking account identifiers were called ids and were represented
+by a corresponding entity's public key.
+
+Now, they are called addresses and are represented by a truncated hash of a
+corresponding entity's public key, prefixed by a 1 byte address version.
+
+Furthermore, the new staking account addresses use the newly added Bech32
+encoding for text serialization with `oasis` as their human readable part
+(HRP) prefix.
+
+The following configuration changes are needed due to this change:
```

Comment:
Move this part into a fragment of tye `cfg`?

Reply:
Done.